capstone/src/main/webapp/signUp.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Gets the information from the textboxes to send to datastore.
+ */
+function sendSignUpInfo() {
+  const name = document.getElementById('new-restaurant-name').value;
+  const cuisine = document.getElementById('new-restaurant-cuisine').value;
+  const story = document.getElementById('new-restaurant-story').value;
+  const phone = document.getElementById('new-restaurant-phone').value;
+  const website = document.getElementById('new-restaurant-website').value;
+
+  // Check if any area doesn't contain any text.
+  if (name == '' || cuisine == '' || story == '' || phone == '' ||
+      website == '') {
+    const pElement = createP('You must fill all text boxes!');
+    const error = document.getElementById('missing-information-error');
+    error.appendChild(pElement);
```

Comment:
Does this error text need to be removed once they fix the issue?

Reply:
So the text remains while the issue remains, however, once the problem is solved and the POST request is made, the backend itself redirects back to the home page.